actionview/test/template/template_test.rb

```diff
@@ -102,6 +102,16 @@ def test_template_does_not_lose_its_source_after_rendering_if_it_does_not_have_a
     assert_equal "Hello", @template.source
   end
 
+  def test_template_only_loads_template_source_once
+    source_loaded = 0
+    source = Class.new do |klass|
+      klass.define_method(:to_s) { source_loaded += 1; "hello" }
```

Comment:
```suggestion
      klass.define_method(:to_s) { source_loaded += 1; +"hello" }
```

Interesting, based on the failing test we actually _must_ return a mutable string here